### DISCUSSION TITLE
When you guys update this will they be able to see that stuff is unblocked and can i get arrested by the school

### DISCUSSION THREAD

**Discussion by Endernates** [3kh0/ext-remover#205] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I'm posting this because I'm kind of lost. And if I can get arrested by the school how likely is it

**Comment by The-Man-Is-Me** [comment]
They will be able to see it, it depends on how active your IT guy is

**Reply by The-Man-Is-Me** [reply]
also, you probably won't get arrested, but it could happen.

**Reply by Theblackmamba824** [reply]
Wait so this can happen with ingot and LTBEEF to

**Reply by ghost** [reply]
@Theblackmamba824 No. It's just a small bookmarklet, not a permanent change to software

**Reply by RuralAnemone** [reply]
> @Theblackmamba824 No. It's just a small bookmarklet, not a permanent change to software

I mean neither is unenrolling so uh

**Comment by ghost** [comment, marked as answer]
You will not get arrested for just the bookmarklet, it's not violating policies in any major way unless you unenroll

**Reply by 6829ASCS** [reply]
Yeah I think getting arrested is probably the lowest punishment you'd get if you're lucky. Probably high chance of getting deported instead? Or maybe even sentenced to death?

**Comment by shinailu** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
